### ISSUE TITLE
Save space by limiting visible splits

### ISSUE THREAD

**Issue by evitalis** [topeterk/HitCounterManager#3]
Would it be possible to have a feature like LiveSplit where you can show only X (configurable) number of splits with the total always visible? That would help keep the overlay uncluttered for viewers.

**Comment by topeterk** [comment]
Sounds interesting.. Lets assume someone wants to have 5 rows of splits shown. What is most important to show to the viewer, the parts that are upcoming or the ones that have been completed? So, for example, shall one see the last 2 completed, the current and the 2 upcoming ones, or better something like 3 completed, the current and the next one?
Or even better make it configurable, like letting choose how many completed, and upcoming splits shall be shown?

**Comment by evitalis** [comment]
I agree with a setting to let the user control it. I would say maybe show the last one or two, and always have the total visible. Not sure how feasible it would be though. I thought about maybe crapping it in OBS but that would require constant manual work.
